tailscale/client.go

```diff
@@ -383,6 +384,15 @@ type (
 		Connectors []string `json:"connectors,omitempty" hujson:"Connectors,omitempty"`
 		Domains    []string `json:"domains,omitempty" hujson:"Domains,omitempty"`
 	}
+
+	Grant struct {
+		Src []string              `json:"src,omitempty" hujson:"Src,omitempty"`
+		Dst []string              `json:"dst,omitempty" hujson:"Dst,omitempty"`
+		IP  []string              `json:"ip,omitempty" hujson:"IP,omitempty"`
+		App map[string][]GrantApp `json:"app,omitempty" hujson:"App,omitempty"`
+	}
+
+	GrantApp map[string][]string
```

Comment:
If you do decide to keep defining the type here instead of using `PeerCapMap`: Be aware that `GrantApp` is any JSON-encodeable value, not just `map[string][]string`.

Reply:
It isn't quite any value, I believe we rule out arrays.